apps/wallet/src/shared/analytics/amplitude.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as amplitude from '@amplitude/analytics-browser';
+import { type UserSession } from '@amplitude/analytics-types';
+import { PersistableStorage } from '@mysten/core';
+
+import { ampli } from './ampli';
+
+const IS_PROD_ENV = process.env.NODE_ENV === 'production';
+
+export const persistableStorage = new PersistableStorage<UserSession>();
+
+export async function initAmplitude() {
+    ampli.load({
+        environment: process.env.NODE_ENV ?? 'development',
+        // Flip this if you'd like to test Amplitude locally
+        disabled: !IS_PROD_ENV,
+        client: {
+            configuration: {
+                cookieStorage: persistableStorage,
+                // This prevents Amplitude from setting its initial
+                // cookies upon initialization of the SDK
+                disableCookies: true,
```

Comment:
Do you need to set this when you set the custom storage?

Reply:
Good thought, let me double check this